pkg/shares/share_splitting_test.go

```diff
@@ -156,10 +156,10 @@ func TestSplitTxs_forTxShares(t *testing.T) {
 						appns.TxNamespace.Bytes(),
 						[]byte{
 							0x0,                 // info byte
-							0x0, 0x0, 0x0, 0x52, // reserved bytes
+							0x0, 0x0, 0x0, 0x4a, // reserved bytes
```

Comment:
[Suggestion, non-blocking] Could we include visual representations illustrating how the data is organized within their respective shares? by visual representation I mean something similar to the ones in [this example](https://github.com/celestiaorg/celestia-app/blob/a4aec0d531bafc6df608206d376b8a431938dc04/pkg/square/builder.go#L194).

Reply:
https://github.com/celestiaorg/celestia-app/issues/1789